test/start_server_test.js

```diff
@@ -74,6 +74,28 @@ suite('startServer', () => {
     })
   });
 
+  suite('proxy', () =>  {
+    test('rewrites directory with proxy', (done) => {
+      let app = startServer({
+        root: __dirname,
+        port: 8080
+      }).then(() => {
+        startServer({
+          port: 5375,
+          root: __dirname,
+          proxyRoot: 'normally-non-existing-path',
+          proxyTarget: 'http://localhost:8080'
+        }).then((app) => {
+          supertest(app)
+          .get('/normally-non-existing-path/bower_components/test-file.txt')
```

Comment:
this path is a little strange, can you just do `/normally-non-existing-path/test-file.txt` or `/normally-non-existing-path/test/test-file.txt`?